pkg/controllers/subnetset/subnetset_webhook.go

```diff
@@ -0,0 +1,112 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package subnetset
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+)
+
+// log is for logging in this package.
+var subnetsetlog = logf.Log.WithName("subnetset-webhook")
+
+var NSXOperatorSA = "system:serviceaccount:vmware-system-nsx:nsx-operator-svc-account"
```

Comment:
nsx-operator container is in NCP pod, which use ncp-svc-account, do we need a new SA here?

Reply:
This is only for local test. Of course we should use ncp sa, thanks, fixed.